reverbePrintemps/spiced
# Use map

# Context

This PR aims to tackle [the map challenge](https://github.com/spiced-academy/honey-web-dev/blob/main/sessions/js-array-methods/challenges-js-array-methods.md#map).

# Changes

- [x] Create a new array with all answers in lower case
- [x] Create a new array with all questions and answers together
- [x] Create a new array with only questions and answers

# How to test

1. Clone repo
2. Checkout this branch
3. Run `npm run test`
4. Verify all tests pass

# TODO


# GIF

![](https://media.giphy.com/media/3ov9jYkVbdGMo6UcG4/giphy-downsized.gif)
